examples/demo/gloo/glsl_sandbox_cube.py

```diff
@@ -2,20 +2,21 @@
 # -*- coding: utf-8 -*-
 """
 A GLSL sandbox application based on the spinning cube. Requires PySide
-or PyQt4.
+or PyQt5.
 """
 
 import numpy as np
 from vispy import app, gloo
 from vispy.io import read_mesh, load_data_file, load_crate
 from vispy.util.transforms import perspective, translate, rotate
 
-# Force using qt and take QtCore+QtGui from backend module
+# Force using qt and take relevant modules from backend
 try:
-    app_object = app.use_app('pyqt4')
+    app_object = app.use_app('pyqt5')
 except Exception:
-    app_object = app.use_app('pyside')
+    app_object = app.use_app('pyside2')
 QtCore = app_object.backend_module.QtCore,
```

Comment:
I'm not sure I understand how this code ever worked with this comma here. Any ideas?